scripts/zkai_attention_kv_proof_size_profile_gate.py

```
@@ -0,0 +1,759 @@
+        "scaled_envelope_bytes_per_score_row",
+        "profile_interpretation",
+        "proof_component_byte_breakdown_status",
+    }
+    if set(axis) != expected_keys:
+        raise AttentionKvProofSizeProfileGateError("scaling diagnostics field drift")
+    expected = build_expected_scaling_diagnostics(single, two_head)
+    if axis != expected:
+        raise AttentionKvProofSizeProfileGateError("scaling diagnostics drift")
+
+
+def build_expected_scaling_diagnostics(single: dict[str, Any], two_head: dict[str, Any]) -> dict[str, Any]:
```

Ruff (0.15.12)

[warning] Unused function argument: `single`

(ARG001)